src/i18n/en-US.json

```diff
@@ -1335,6 +1335,10 @@
   "tooltipConversationAllDevicesVerified": "All device fingerprints verified (Proteus)",
   "tooltipConversationAllE2EIVerified": "All devices are verified (end-to-end identity). [link]Learn more[/link]",
   "tooltipConversationAllE2EIVerifiedShort": "All devices verified (end-to-end identity)",
+  "tooltipConversationAllE2EINewDeviceAdded": "This conversation is no longer verified, as [bold]{{user}}[/bold] uses at least one device without a valid end-to-end identity certificate.",
+  "tooltipConversationAllE2EINewUserAdded": "This conversation is no longer verified, as [bold]{{user}}[/bold] uses at least one device without a valid end-to-end identity certificate.",
+  "tooltipConversationAllE2EICertificateExpired": "This conversation is no longer verified, as [bold]{{user}}[/bold] uses at least one devices with an expired end-to-end identity certificate.",
+  "tooltipConversationAllE2EICertificateRevoked": "This conversation is no longer verified, as a team admin revoked the end-to-end identity certificate for at least one of [bold]{{user}}[/bold] devices. [link]Learn more[/link]",
```

Comment:
**question**: should it include tooltip in the name? I don't think we render it in a tooltip 🤔